athena-gcs/src/main/java/com/amazonaws/athena/connectors/gcs/common/PartitionColumnData.java

```diff
@@ -0,0 +1,59 @@
+/*-
+ * #%L
+ * athena-gcs
+ * %%
+ * Copyright (C) 2019 - 2023 Amazon Web Services
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+package com.amazonaws.athena.connectors.gcs.common;
+
+import static java.util.Objects.requireNonNull;
+
+/**
+ * Used to get Storage Partition
+ */
+public class PartitionColumnData
+{
+    private final String columnName;
+    private final String columnType;
+    private final String columnValue;
+
+    public PartitionColumnData(String columnName, String columnType, String columnValue)
+    {
+        this.columnName = requireNonNull(columnName, "columnName was null");
+        this.columnType = requireNonNull(columnType, "columnType was null");
+        this.columnValue = requireNonNull(columnValue, "columnValue was null");
+    }
+
+    public String getColumnName()
+    {
+        return columnName;
+    }
+
+    public String getColumnValue()
+    {
+        return columnValue;
+    }
```

Comment:
There's no need for these getters, just make the member variables public since they are final strings. This means that they can't be modified after construction, so there's no point in making them private and then providing getters.